### PR TITLE
feature(app): implement app to avoid CORS

### PR DESCRIPTION
**Pull request by wd15**
Address #903

Currently any data source such as Google Drive is inaccessible to
PFHub due to the CORS issue. This app will allow access to Google
Drive as well as other CORS restricted data repositories.

 - The app is currently run from my personal Google Apps account. This
   will need to be addressed going forward.

- To review, check that data is loaded on all the Plotly plots and http://random-cat-1015.surge.sh/simulations/display/?sim=Spinodal_c loads since it's and example of using Google Drive.

These changes can be [viewed live][live-site].

**Remember to tear down the [live site][live-site] when merging
or closing this pull request.**

[live-site]: http://random-cat-1015.surge.sh

---
This change is [Reviewable](https://reviewable.io/reviews/usnistgov/pfhub/1015)

### CODE REVIEW

**Comment by dschwen** [comment]
Hm, maybe I'm missing the point, but I've used the Google Drive API before and had no CORS issues.

**Comment by wd15** [comment]
@dschwen, is that from the client-side or server-side?

**Comment by dschwen** [comment]
From the client side. It was a web app that allowed users to store and load
data from their Google Drive accounts. It even included logging in. I
assume the data we use will be publicly accessible anyways.

**Comment by dschwen** [comment]
This one: https://github.com/dschwen/jszip :-)

**Comment by wd15** [comment]
Wow, that looks awesome! Whenever I try with PFHub I get a CORS domain issue with Google Drive. I may well be missing something though or need to modify the URL in some way or fetch the data with a different request. Can you point me to the bit of code that actually does it?

**Comment by dschwen** [comment]
https://github.com/dschwen/jszip/blob/11416b0fbbd9cf2ef4eca3a051eb2859762918ab/storage.js#L273

**Comment by wd15** [comment]
@dschwen, you are a man of many talents! This is very useful, but I'm going to go ahead with this PR anyway. One reason is that the Plotly JS is currently fetching the data files so requires a URL so I can't do anything fancy. Also, I actually want to start processing data via an app with Python just because than JS it's easier and I can get bigger data files. Also, I just want to get this working and I can refactor later.

**Comment by dschwen** [comment]
Makes sense

File: _app/.gcloudignore [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# This file specifies files that are *not* uploaded to Google Cloud Platform
```
Comment (tkphd) [inline _app/.gcloudignore]: Does this file need to be included?

File: _app/README.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# Play with FastAPI
```
Comment (tkphd) [inline _app/README.md]: Sure looks like we're using it...

File: _data/simulations/Spinodal_c/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    container_url: ''
+  hardware:
+    cpu_architecture: x86_64
+    acc_architecture: non-gpu
+    parallel_model: serial
+    clock_rate: '2.4'
+    cores: '1'
+    nodes: '1'
+benchmark:
+  id: 1a
```
Comment (tkphd) [inline _data/simulations/Spinodal_c/meta.yaml]: 1c

**Pull request merged**